src/test/javascript/e2e/cypress/integration/util/login.ts

```diff
@@ -1,16 +1,13 @@
+import * as navBarPage from "./nav-bar";
+
 export function login(username = "admin", password = "admin") {
-    return cy.request({
-        url: "/oauthserver/oauth/token",
-        method: "POST",
-        headers: {
-            'Content-Type': 'application/x-www-form-urlencoded',
-        },
-        body: `username=${username}&password=${password}&grant_type=password`
-    }).then(res => {
-        let token = res.body;
-        const expiredAt = new Date();
-        expiredAt.setSeconds(expiredAt.getSeconds() + token.expires_in);
-        token.expires_at = expiredAt.getTime();
-        cy.setCookie("oAtkn", JSON.stringify(token));
-    });
+    cy.visit('./');
```

Comment:
This is the "right" way to log in, but it is slower. Previous version intentionally sends a request instead of signing in via the UI to speed up the tests that do not verify the Log In functionality.
(I do not see this change as an improvement)

Reply:
A few ideas here:
- the old way of logging in no longer worked: without Zuul proxy a session needs to be used, not a token. This way makes e2e logins behave as they would in the frontend, and do not rely on the backend implementation of tokens
- decouple tests
- I started using the cy.session API to speed up logins, but it is in beta and proved not to be reliable yet.

as a workaround, I could login once per test file and preserve the `SESSION` cookie. It would make the coupling between tests slightly stronger though.

Reply:
>without Zuul proxy a session needs to be used, not a token
Okay, I didn't realize this was the case, I thought it was an "unprovoked" change.

No objections then. I didn't run the tests yet — maybe they are fast enough as-is with the current approach.

Reply:
I just checked; it used to be 4:50 min, it’s 4:35 min now - perhaps because of some source data tests that had been commented out though.